pages/deploy/deploy_aws.md

```diff
@@ -518,11 +526,12 @@ check the [UserGuide](https://github.com/data-dot-all/dataall/blob/main/UserGuid
 | catalog         | None                                                | Central catalog of data items. In this module a glossary of terms is defined.                                              |
 | feed            | None                                                | S3 Bucket and Glue database construct to store data in data.all                                                            |
 | vote            | catalog                                             | S3 Bucket and Glue database construct to store data in data.all                                                            |
```

Comment:
We should have an entry for datasets_base independent from s3_datasets no?

Reply:
I got confused, the issue is that s3_datasets does not depend on the s3_datasets_shares anymore (that was one of the main points of the refactoring)